### DISCUSSION TITLE
Pretrained models MICCAI surgtoolloc

### DISCUSSION THREAD

**Discussion by pejokoe** [Project-MONAI/MONAI#6875] [Q&A]
As suggested by the title, I am wondering about the availability of fully trained models for the classification and detection components of the solution for the MICCAI surgtoolloc competition, found in /tutorials/tree/main/competitions/surgtoolloc. 
I would like to take first experimental steps towards domain adapting the model to neuro surgery but lack the resources to perform a trainig myself.

**Comment by KumoLiu** [comment]
Hi @pejokoe, thanks for your interest here.

We didn't provide the pre-trained weights for this challenge, but you can find some similar model weights here:
- https://github.com/Project-MONAI/model-zoo/tree/dev/models/endoscopic_inbody_classification
- https://github.com/Project-MONAI/model-zoo/tree/dev/models/endoscopic_tool_segmentation
- https://github.com/Project-MONAI/model-zoo/tree/dev/models/lung_nodule_ct_detection

MONAI Model Zoo hosts a collection of medical imaging models in the [MONAI Bundle](https://docs.monai.io/en/latest/bundle_intro.html) format. Also, welcome to contribute your models.

I hope it helps, thanks!
